examples/demo-apps/android/LlamaDemo/app/src/main/AndroidManifest.xml

```diff
@@ -47,6 +47,15 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <activity
+            android:name=".Benchmarking"
+            android:exported="true">
+            <intent-filter>
+                <action android:name="com.example.executorchllamademo.BENCHMARK" />
```

Comment:
1. Can we name it to be something more generic, e.g. llm benchmark runner
2. Can we later move the entire app under `executorch/extension/llm` as an extension for llm benchmarking?

Reply:
2) is not a blocker for this PR. Once you addressed 1) this PR should be ready to go

Reply:
Fixed 1 now. Working on moving it out of llamademoapp and use a separate app (for generic as well)